### ISSUE TITLE
Unable to add users to new groups on emoncms.org

### ISSUE THREAD

**Issue by pb66** [emoncms/group#37]
On emoncms.org I am unable to add a user to a new group as reported on the OEM forum in the ["One device - different users"](https://community.openenergymonitor.org/t/one-device-different-users/7541/6?u=pb66) thread. Just tried it again and it is still a current issue, although oddly, I have noticed that I can add the same user to a previously created test group that I created way back when the groups module first appeared on emoncms.org. So I am currently able to add a user to an old existing group, but not to a newly created group. 

![image](https://user-images.githubusercontent.com/5606042/41052877-f1898112-69b1-11e8-8483-02187db9163c.png)

**Comment by pb66** [comment]
Ok... Some more info for you, I tried creating a new group and added a user to it to test if it was possibly just 1 corrupt group that was the issue and it did allow me to add another user, BUT, what I noticed was that all the working groups have me as a default user and admin, and that there is no way to delete me. The faulty group has no members, not even me.

So I tried "create new user" on another new good group and that resulted in the following "undefined error" 

![image](https://user-images.githubusercontent.com/5606042/41055896-5e7ef290-69ba-11e8-8550-42c2aa640d69.png)

and a second click of the button told me that the user already exists

![image](https://user-images.githubusercontent.com/5606042/41055937-81c652f2-69ba-11e8-8589-29aab0e292e1.png)

but when I closed that modal box, I was no longer a member of that group, the group was now empty.

This is easy to reproduce by 
1. create new group (check you are admin) (ps I used a numeric first char, if that matters eg 10home)
2. create new user
3. after the errors and modal screen closed you will no longer be admin for that group, it will be empty.

I have not tested this on a self hosted, only on emoncms.org so I have no way of checking the tables or logs etc.

Plus I do not know who will get notifications so I will ping @cagabi  and also @TrystanLea since it is the emoncms.org implementation.

**Comment by cagabi** [comment]
Hi Paul,
I am testing in my self hosted up to date installation and everything is working ok. I am waiting for my email verification in emoncms.org to try it there but I am afraid I cannot help on that one.

**Comment by cagabi** [comment]
Ah, I get all notifications for issues in this repository :)

**Comment by cagabi** [comment]
Hi again, tried in emoncms.org and you are right it's broken. I got the same pop ups, first saying "undefined" then saying "Name already exists" and now the page loads but the group seems to be empty

**Comment by pb66** [comment]
So should this be moved to the emoncmsorg repo? or will @TrystanLea  (or @emrysr) pick this up here ?

We don't even know if emoncmsorg is running the latest group module or if it's modded in anyway for emoncmsorg specifically.

**Comment by cagabi** [comment]
Good point. Let's wait and see if we hear from them soon and if not open an issue in emoncms.org
